src/dealDashboard/dealDiscussion/discussionThread/discussionThread.ts

```diff
@@ -340,6 +350,6 @@ export class DiscussionThread {
   }
 
   private navigateTo() {
-    this.discussionId = null;
+    this.discussionId = "";
```

Comment:
I assume the empty string denotes the discussion is not selected. I would recommend we use `undefined` for when a discussion is not selected. An empty string might be confusing. (you can also remove the default empty string from the class property)

_Optional_

Reply:
done